app/mailers/notify_mailer.rb

```diff
@@ -35,9 +35,11 @@ def new_mention_email
 
     @mentioner = User.find(@mention.mentionable.user_id)
     @mentionable = @mention.mentionable
+    @mentionable_type = MentionDecorator.new(@mention).formatted_mentionable_type
```

Comment:
**non blocking**: you can also use `@mention.decorate` instead of `MentionDecorator.new(@mention)`